darts/models/forecasting/forecasting_model.py

```diff
@@ -896,6 +899,63 @@ def model_params(self) -> dict:
             self._model_params if hasattr(self, "_model_params") else self._model_call
         )
 
+    def save_model(
```

Comment:
I'd have preferred to call them `save()` and `load()`, but maybe it is too late...
We could perhaps deprecate these names with a warning message, and mention in the future only `save()` and `load()` will be available. WDYT?

Reply:
Yeah, I'd be in favor of doing this. I think as long as it is only models that we can call these methods on we don't need the "_model" suffix.